ios/FluentUI/SegmentedControl/SegmentedControl.swift

```diff
@@ -194,6 +203,13 @@ open class SegmentedControl: UIControl {
         view.layer.cornerCurve = .continuous
         return view
     }()
+    private let pillMaskedLabelLayerView: UIView = {
+        let view = UIView()
+        view.layer.cornerCurve = .continuous
+        return view
+    }()
+    private var pillMaskedLabels = [UILabel]()
+    private var pillMaskedLabelsConstraints = [[NSLayoutConstraint]]()
```

Comment:
we kind of talked offline but is it necessary to hold on to these constraints? easy way to test it is to do memory profiling....